### DISCUSSION TITLE
Katex Server Side

### DISCUSSION THREAD

**Discussion by agungizzulhaq** [KaTeX/KaTeX#2440] [Q&A] [answered]
Is there a detailed guide on how to use KaTeX in server-side?

**Comment by edemaine** [comment, marked as answer]
For starters, I would read https://katex.org/docs/node.html and https://katex.org/docs/api.html

After that, let us know if you have any more specific questions?

**Reply by agungizzulhaq** [reply]
Thanks. I'll learn about Node.js first.

**Reply by agungizzulhaq** [reply]
I have tried it. But, i don't know how to render a paragraph that contains some text. For example:
"Let $a \in \mathbb R$."
Do you know how to do that?

**Reply by k4b7** [reply]
@agungizzulhaq the auto-render extension can help with this. https://katex.org/docs/autorender.html
